Add config flow to dwd_weather_warnings (#26914)
home-assistant/home-assistant.github.io: source/_integrations/dwd_weather_warnings.markdown

---
title: Deutscher Wetterdienst (DWD) Weather Warnings
description: Instructions on how to integrate Deutscher Wetterdienst weather warnings into Home Assistant.
ha_category:
  - Weather
ha_config_flow: true
ha_release: 0.51
ha_iot_class: Cloud Polling
ha_domain: dwd_weather_warnings
ha_codeowners:
  - '@runningman84'
  - '@stephan192'
  - '@Hummel95'
  - '@andarotajo'
ha_platforms:
  - sensor
ha_integration_type: integration
---

> The `dwd_weather_warnings` integration uses the [Deutscher Wetterdienst (DWD)](https://www.dwd.de) as a source for current and advance weather warnings. The configured sensor checks for data every 15 minutes.

Reviewer: ```suggestion
The Deutscher Wetterdienst Weather Warnings integration uses the [Deutscher Wetterdienst (DWD)](https://www.dwd.de) as a source for current and advance weather warnings. The configured sensor checks for data every 15 minutes.
```

{% include integrations/config_flow.md %}

{% configuration_basic %}
Warncell ID or name:

>   description: Identifier of the region. It can either be a so called `warncell ID` (integer) or a `warncell name` (string). It is heavily advised to use `warncell ID` because `warncell name` is not unique in some cases. A list of valid warncell IDs and names can be found [here](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv.html). Some of the warncells are outdated but still listed. If the setup fails, search the list for a similar sounding warncell. In case the `warncell name` is not unique, `" (not unique used ID)!"` will be added to the reported `region_name`. Setting this field is required.

Reviewer: ```suggestion
  description: Identifier of the region. It can be a warncell ID (integer) or a warncell name. It is heavily advised to use warncell ID because a warncell name is sometimes not unique. A list of valid warncell IDs and names can be found [here](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv.html). Some of the warncells are outdated but still listed. If the setup fails, search the list for a similar sounding warncell. If the warncell name is not unique, `" (not unique used ID)!"` will be added to the reported `region_name`. Setting this field is required.
```
